lib/cadet_web/controllers/sourcecast_controller.ex

```diff
@@ -62,6 +85,12 @@ defmodule CadetWeb.SourcecastController do
     security([%{JWT: []}])
 
     parameters do
+      public(
+        :body,
+        :boolean,
+        "Uploads as public sourcecast when 'public' is specified regardless of truthy or falsy"
```

Comment:
Please don't do this, check whether it is `true` or `false` else 400

Reply:
Ping

Reply:
Okay! but the public option will only come in after we do the abuse-prevention upgrade, plus the frontend is not using this at the moment cuz we realized that sourcecast is behind log in and we do not plan to have any sourcecast that are shared among all courses.